README.md

```diff
@@ -19,7 +19,7 @@ The authors take no responsibility for any loss of digital assets or other damag
 
 ## Getting Started
 
-Running _perun-eth-demo_ requires a working Go distribution (version 1.14 or higher).   
+Running _perun-eth-demo_ requires a working Go distribution (version 1.14 or higher) and [ganache-cli](https://github.com/trufflesuite/ganache-cli) installed.   
```

Comment:
It should also work with *geth* but we normally test it with *ganache-cli* since it is easier to launch.  
Real eth chains like *ropsten* connected through *Infura* e.g. is ok too, so we should probably just write that you need access to a trustworthy Ethereum Node but in this example we assume *ganache-cli* for ease of use.

Reply:
i agree, it should work with other backends. however, in the "Example Walkthrough" you explicitly refer to ganache-cli. to make the walkthrough as user friendly as possible, i suggest to explicitly mention that you assume ganache-cli installed, otherwise the walkthrough will not work.

Reply:
Maybe something like this?
```suggestion
Running _perun-eth-demo_ requires a working Go distribution (version 1.14 or higher). Optionally, [ganache-cli](https://github.com/trufflesuite/ganache-cli) needs to be installed for following the walkthrough. _perun-eth-demo_ works with any ethereum node.
```

Reply:
looks good to me, i adopted this change suggestion to the code base